Running with no args produces panic

Using the latest v0.17.0-alpha4 release if you run headscale with no command line args it produces an error rather than the help text as in previous versions:

$ ./headscale
panic: runtime error: index out of range [1] with length 1

goroutine 1 [running]:
github.com/juanfont/headscale/cmd/headscale/cli.init.8()
	/home/runner/work/headscale/headscale/cmd/headscale/cli/root.go:18 +0x26f

--- a/cmd/headscale/cli/root.go
+++ b/cmd/headscale/cli/root.go
@@ -15,7 +15,7 @@ import (
 var cfgFile string = ""
 
 func init() {
-	if len(os.Args) > 1 && os.Args[1] == "version" || os.Args[1] == "mockoidc" {
+	if len(os.Args) > 1 && (os.Args[1] == "version" || os.Args[1] == "mockoidc") {
 		return
 	}
 

